### ISSUE TITLE
[Bug]Unable to create components

### ISSUE THREAD

**Issue by nefudev** [ltdrdata/ComfyUI-Workflow-Component#24]
I was able to create components without any problems, but after updating ComfyUI and all extensions today, I have the following problems.

1. when I create a component, the inputs are corrupted
2. component cannot be saved with "Export As Component

## Tried
- When I load a previously created component and re-create it, the inputs are corrupted.
- Reverted to October 1 and 5 versions, but no improvement.
- When I revert to the previous version, problem 2 occurs.
- When I revert to the latest version, problem 2 still remains.
- The server and browser are closed and all browser caches are deleted, but the same thing happens.
- The previously created components are working fine.

The hashes of ComfyUI and extensions currently in use are as follows(All the latest versions at this time)

ComfyUI:1f2f4eaa6f04660a7df7d71eede3118d09b1c2c2
ComfyUI-Workflow-Component:9e8ccf5fa163cf2febabffff40989f77e8c51e62
ComfyUI-Impact-Pack: 12b5b1a5709d7ef11b384ee239a1575572f4d116

![img_0001 3](https://github.com/ltdrdata/ComfyUI-Workflow-Component/assets/132149510/84b8a66f-8b80-4262-9856-64f9c0e3a039)

**Comment by ltdrdata** [comment]
I updated Workflow Component for latest ComfyUI.

One common mistake when recreating a workflow is ensuring that the name of input interfaces of components do not have duplicates.
And in cases where it shows as "BROKEN," it is necessary to check the terminal messages. This typically happens when using an not installed custom node or error node inside component.

BTW,
I am currently using the Workflow of the existing structure and identifying areas for improvement. The biggest issue is the backward compatibility problem when the structure of a custom node changes. I am currently considering how to address this issue.

I hope to see improvements in such issues in the workflow component that will be based on PR931 in the future.

**Comment by MortalSage** [comment]
I had the same issue, until I updated just now. After the update I didn't get broken components anymore, but the export button seems broken. Nothing happened at all. 
Tried to locate the issue by reducing component complexity, I found that text-inputs prevent me from exporting the component. For example I am not allowed to make denoise or prompt an input anymore. 
If I remove all text-inputs, I can export and use components again. Also, old custom-nodes with text-inputs in it are still working.

**Comment by nefudev** [comment]
Thanks for the reply.

I'll double check and note what I noticed.
-  A search of the nodes created did not reveal any duplicate input interface names.
- I ran "Install Missing Custom Nodes" in comfyUI Manerger and found nothing.
- Workflows created with comfyUI's "Load Default" can be saved as components.
- Adding inputs to "Load Checkpoint" in the above workflow prevents saving.
- I can't save after clearing everything and placing a new node

The following error was confirmed when the corrupt node was read and viewed.
``` bash
[Workflow-Component] '## workflow102 [b7280c]' is broken. Maybe there are missing nodes. (INFO: 31)
Traceback (most recent call last):
  File "<ComfyUI_dir>\custom_nodes\ComfyUI-Workflow-Component\workflow_component\component_loader.py", line 209, in get_input_types_dynamic
    build_input_types(i, input_mapping, input_types, node, node_config_map, spec_map)
  File "<ComfyUI_dir>\custom_nodes\ComfyUI-Workflow-Component\workflow_component\component_loader.py", line 336, in build_input_types
    if node_config_map[i][1] is not None or node_config_map[i][2] is not None or node_config_map[i][3] is not None or node_config_map[i][4] is not None:
KeyError: 31
```

The structure of workflow102 is described below.(The generated nodes are the same as in the first image) 
![img_0001 4](https://github.com/ltdrdata/ComfyUI-Workflow-Component/assets/132149510/d0bb1f60-dc65-41f6-ba9d-1bada249f9d9)

**Comment by R-ES-H** [comment]
Same thing here, I can't export as component if any widget is converted to an input

```
[Workflow-Component] '## NEXUS [614180]' is broken. Maybe there are missing nodes. (INFO: 5300)
Traceback (most recent call last):
  File "E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Workflow-Component\workflow_component\component_loader.py", line 209, in get_input_types_dynamic
    build_input_types(i, input_mapping, input_types, node, node_config_map, spec_map)
  File "E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Workflow-Component\workflow_component\component_loader.py", line 336, in build_input_types
    if node_config_map[i][1] is not None or node_config_map[i][2] is not None or node_config_map[i][3] is not None or node_config_map[i][4] is not None:
KeyError: 5300
```

**Comment by ltdrdata** [comment]
With the update to ComfyUI, I made a compatibility patch, but then another patch changed it back to the old way, causing issues. Now, it should work well again.

**Comment by ltdrdata** [comment]
My mistake.

**Comment by ltdrdata** [comment]
This is fixed on b2b11b191be6b15eb382a6ab0c0339dd3d42aa60

V0.43.2
